less/home-page.less

```diff
@@ -1,75 +1,152 @@
 // Home Content
-
 .home {
 
-  .intro {
-
-    h2 {
-      padding: 15px;
-    }
+  @media @phone {
 
-    p {
-      padding:0 15px;
-    }
+    align-items: center;
+    justify-content: center;
+    width      : 90%;
+    margin     : 0 auto;
+    
   
   }
 
-  .space-images {
-    display: flex;
-    flex-wrap: wrap;
-    width: 100%;
+}
 
-    img {
-      width: 28%;
-      margin: 20px 2.5%;
-      border-radius: 20px;
-      height: 100%;
-    }
+.intro {
+  .img-header{
+    @media @phone {
+      align-items    : center;
+      justify-content: center;
+      width          : 100%;
+      margin         : 0 auto;
+  }
   }
 
-  .visit-planets {
+  h2 {
     padding: 15px;
+  }
 
-    h3 {
-      margin-top: 11px;
-    }
+  p {
+    padding: 0 15px;
+  }
 
-    p {
-      margin-top: 13px;
-    }
+
+  @media @phone {
+    display       : flex;
+    flex-direction: column;
+    align-items   : center;
+   
+  }
+
+
+}
+
+.space-images {
+  display  : flex;
+  flex-wrap: wrap;
+  width    : 100%;
+
+  @media @phone {
+    display       : flex;
+    flex-direction: column;
+    align-items   : center;
+  }
+
+  img {
+    width        : 28%;
+    margin       : 20px 2.5%;
+    border-radius: 20px;
+    height       : 100%;
+    @media @phone {
+      align-items    : center;
+      justify-content: center;
+      width          : 100%;
+      margin         : 0 auto;
+  }
   }
 
-  .walk {
-    margin-top: 60px;
-    border-top: 1px solid @main-border-color;
-    padding: 50px 60px;
-    display: flex;
+  @media @phone {
+    align-items    : center;
     justify-content: center;
+    width          : 100%;
+    margin         : 0 auto;
+}
+}
+
+.visit-planets {
+  padding: 15px;
 
-    .walk-text {
-      width: 50%;
+  h3 {
+    margin-top: 11px;
+  }
 
-      h3 {
-        margin-bottom: 10px;
-      }
+  p {
+    margin-top: 13px;
+  }
 
-      p {
-        margin-bottom: 26px;
-      }
+  
+}
+
+.walk {
+  margin-top     : 60px;
+  border-top     : 1px solid @main-border-color;
+  padding        : 50px 60px;
+  display        : flex;
+  justify-content: center;
+
+  .walk-text {
+    width: 50%;
+
+    h3 {
+      margin-bottom: 10px;
+    }
 
-      .btn {
-      }
+    p {
+      margin-bottom: 26px;
     }
 
-    .walk-img {
-      width: 46%;
-      margin-left: 4%;
-      
-      img {
-        width: 100%;
-        height: auto;
-        border-radius: 10px;
-      }
+    .signup-btn {
+      .button-decoration(@btn-bg-color, @porcelain, @btn-size, 10%);
+      padding: 1% 4%;
+      border: none;
+      @media @phone {
+        align-items    : center;
+        justify-content: center;
+        width          : 100%;
+        margin         : 0 auto;
+        
+    }
     }
   }
-}// home
\ No newline at end of file
+  @media @phone {
+    align-items    : center;
+    justify-content: center;
+    flex-wrap: wrap;
```

Comment:
For the mobile view in the Walk section.  I see that what you implemented was wrap and then set the order of the img to -1.  An easier solution might be column reverse.